views/header.php

```diff
@@ -38,6 +38,16 @@ function display($results) {
                                     $nodeStatus = $node['status']['state'];
                                 }
                             }
+
+                            // VMs only - Node shows as critical if conditions are met
+                            if (strpos($node_name, "vm") !== false) {
+                                if (($node["branch_name"] != "prod") or ($node["personality"] != "nubesvms")) {
+                                    if ($node['status']['state'] == "uninstantiated") {
```

Comment:
Are you sure this is the right way round?
Surely the node should only be critical if it is instantiated.